diff --git a/node_modules/react-three-fiber/types/src/three-types.d.ts b/node_modules/react-three-fiber/types/src/three-types.d.ts
index 212b144..ba355a9 100644
--- a/node_modules/react-three-fiber/types/src/three-types.d.ts
+++ b/node_modules/react-three-fiber/types/src/three-types.d.ts
@@ -61,7 +61,7 @@ export declare namespace ReactThreeFiber {
 declare global {
   namespace JSX {
     interface IntrinsicElements {
-      audio: ReactThreeFiber.Object3DNode<THREE.Audio, typeof THREE.Audio>
+      audio_: ReactThreeFiber.Object3DNode<THREE.Audio, typeof THREE.Audio>
       audioListener: ReactThreeFiber.Object3DNode<THREE.AudioListener, typeof THREE.AudioListener>
       positionalAudio: ReactThreeFiber.Object3DNode<THREE.PositionalAudio, typeof THREE.PositionalAudio>
       mesh: ReactThreeFiber.Object3DNode<THREE.Mesh, typeof THREE.Mesh>
@@ -73,7 +73,7 @@ declare global {
       bone: ReactThreeFiber.Object3DNode<THREE.Bone, typeof THREE.Bone>
       lineSegments: ReactThreeFiber.Object3DNode<THREE.LineSegments, typeof THREE.LineSegments>
       lineLoop: ReactThreeFiber.Object3DNode<THREE.LineLoop, typeof THREE.LineLoop>
-      line: ReactThreeFiber.Object3DNode<THREE.Line, typeof THREE.Line>
+      line_: ReactThreeFiber.Object3DNode<THREE.Line, typeof THREE.Line>
       points: ReactThreeFiber.Object3DNode<THREE.Points, typeof THREE.Points>
       group: ReactThreeFiber.Object3DNode<THREE.Group, typeof THREE.Group>
       immediateRenderObject: ReactThreeFiber.Object3DNode<
